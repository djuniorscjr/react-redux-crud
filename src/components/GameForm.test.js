import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import GameForm from './GameForm';

test('GameForm component should render as expected', () => {
	const component = shallow(<GameForm />);
	const tree = toJson(component);
	expect(tree).toMatchSnapshot();
});
