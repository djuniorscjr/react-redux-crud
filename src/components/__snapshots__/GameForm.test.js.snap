// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameForm component should render as expected 1`] = `
<div>
  <form
    className="ui form"
    onSubmit={[Function]}
  >
    <h2>
      Add new game
    </h2>
    <div
      className="field"
    >
      <label
        htmlFor="title"
      >
        Title
      </label>
      <input
        id="title"
        name="title"
        onChange={[Function]}
        value=""
      />
      <span />
    </div>
    <div
      className="field"
    >
      <label
        htmlFor="cover"
      >
        Cover URL
      </label>
      <input
        id="cover"
        name="cover"
        onChange={[Function]}
        value=""
      />
      <span />
    </div>
    <div
      className="field"
    />
    <div
      className="field"
    >
      <button
        className="ui primary button"
      >
        Save
      </button>
    </div>
  </form>
</div>
`;
